### ISSUE TITLE
Update accessibility checklist vignette

### ISSUE THREAD

**Issue by matt-dray** [best-practice-and-impact/aftables#88]
Update [the vignette](https://co-analysis.github.io/a11ytables/articles/checklist.html) that compares {a11ytables} to the [accessibility checklist](https://analysisfunction.civilservice.gov.uk/policy-store/making-spreadsheets-accessible-a-brief-checklist-of-the-basics/).

Via Hannah: 

>I have removed “tagging worksheet headings and subheadings” in spreadsheets from “essential” on the spreadsheet checklist (and within the main spreadsheet guidance).

I think there be some new items in there too, like 'columns should be a sensible width' under the 'Tables' section.
